frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -67,7 +70,17 @@ export const infiniteListLogic = kea<infiniteListLogicType>({
     key: (props) => `${props.taxonomicFilterLogicKey}-${props.listGroupType}`,
 
     connect: (props: InfiniteListLogicProps) => ({
-        values: [taxonomicFilterLogic(props), ['searchQuery', 'value', 'groupType', 'taxonomicGroups']],
+        // TODO: had to connect FF to get the model loaded for filtering
```

Comment:
@mariusandra specifically here, I think this is fine to do, but wanted another set of eyes here!

Reply:
Well, this works... but my main concern is that this `lib/component` now pulls in a lot of deeply nested logics just to display a universal dropdown. 🤔 

Pulling feature flagLogic is probably fine, but I'm not 100% convinced of the other ones. Perhaps they could be passed in as `props` to the logic, like `appScenes` are originally passed to `sceneLogic`.

Reply:
Makes sense! I actually don't need these at all in here, just need to make sure they're mounted, so I moved this to `UniversalSearchPopup` instead, keeping this component clean and blind.

https://github.com/PostHog/posthog/pull/9299/commits/c647b6a46741eb40f61ef37cfbb4fd0cb597c934

Is there a better way to ensure a logic is mounted? I don't face this problem with others because they're models that are mounted on app start